text/0002-split-coinstake-transaction/0002-split-coinstake-transaction.md

```diff
@@ -37,21 +35,23 @@ Only the coinbase `vin[0]` has been used to carry meta-data (e.g. signaling P2SH
 Splitting the coinstake transaction into a *monetary creation* and a *coin-age consumption* transaction, has the side effect of creating small value outputs.
 Because those outputs are very unlikely to mint new blocks, node implementations are adviced to join them with a future *coin-age consumption* transaction.
 
-The minter is awarded a budget of 1 KB for the *coin-age consumption* transaction in order to adjust their UTXO table as they see fit.
-If the *coin-age consumption* transaction is greater than 1 KB, it will have to include the standard fee less 1 KB.
+The minter is currently awarded a budget of 1 KB for the coinstake transaction, and will continue to have this power.  If a *coin-age consumption* transaction is greater than 1 KB, it will have to include the standard fee less 1 KB.
 
 The diagram below illustrates the block layout with a splitted coinstake transaction.
 
 ![splitted block layout](split.png)
 
 ## Advantages
 
-* Paving the road for other improvements like multi-signature minting.
-* Limiting the power of minters to spam the blockchain.
+* Required for multi-signature minting, as described in [RFC-0003](../0003-multisig-minting/0003-multisig-minting.md)
+* While there is no known advantage for aligning the transaction structure with Bitcoin's, it should be taken as a step toward simplification, which can foster innovation.
 
 ## Drawbacks
 
-* Hard fork
+* Dependency on [RFC-0004](../0004-remove-transaction-timestamp/0004-remove-transaction-timestamp.md).
```

Comment:
Why is this a drawback?

Reply:
Dependencies are drawbacks because a) they require additional work to implement and b) they inherit the drawbacks of their dependencies.  If rfc0002 could be implemented independently of rfc0004 without losing any benefits or gaining any additional drawbacks, it would logically be a 'better', or at least more flexible, proposal wouldn't it?

Reply:
That is like saying it would be better if both rfc0002 and rfc0004 where merged into one rfc?

Reply:
It would indeed be better from the perspective of rfc0002 because it would gain advantages without gaining disadvantages.  However, it would make rfc0004 much harder to implement.  Modularity allows for separate analysis and pro-con weighing.

Ultimately, it is just a convenient convention to explicitly identify dependencies in this way.  With the way the rfc numbers are labeled, I think it helps to clarify things.

Reply:
ok, fine.

Reply:
I would like to actually seriously consider this question.  When forming the next push, it seems like RFC0002 devolves in its drawbacks to basically having none, assuming RFC0004 is implemented.  What I mean by this, is that if we are able to do the hardfork at the same time as RFC0004, then there are literally no drawbacks.  There is absolutely no reason not to do RFC0002 if we are doing RFC0004.

This brings up the question of why we shouldn't merge them into one?  The only reason I can come up with is that it is too complicated to explain it all at once, or that we may not want to do the required work to do them both at once.  I am trying to understand the convention here for when we should split RFCs into parts vrs when we should merge them, especially in a case like this where after the first RFC is implemented, there is nothing to lose and some to gain by doing the next one as well.

Reply:
I don't think we need to list dependencies as drawbacks. There could be a section about prerequisites or something though, because the summary should not be the only place to mention that (I think we should be able to skip the summary and still have all the information).

> There is absolutely no reason not to do RFC0002 if we are doing RFC0004.

Yes there are. RFC 2 is a big protocol change. It means there are risks and it increases code complexity, mostly because we have to handle both the old and new systems.

If a merge could be considered it would be between RFCs 2 and 3 since 2's main purpose it to enable 3. But I'd rather keep them separate, it wouldn't really help the discussion.